frontend/src/Wrapper.tsx

```diff
@@ -2,8 +2,10 @@ import Box from '@mui/material/Box'
 import { useTheme } from '@mui/material/styles'
 import Toolbar from '@mui/material/Toolbar'
 import { useGetUiConfig } from 'actions/uiConfig'
+import Cookies from 'js-cookie'
```

Comment:
I know the library uses `Cookies` in their examples, but can we use `cookies` as it then follows our naming conventions?